api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+{
+        "swagger": "2.0",
+        "info": {
+                "title": "Cilium API",
+                "description": "Cilium",
+                "version": "unversioned"
+        },
+        "schemes": [
+                "http"
+	],
+        "x-schemes": [
+		"unix"
```

Comment:
x-schemes is a go-swagger extension to allow unix domain sockets in the generated code.